README.md

```diff
@@ -32,33 +33,16 @@
   - [❓ FAQ ](#-faq-)
   - [📝 License ](#-license-)
 
-# 📖 WebPack Template <a name="about-project"></a>
-
-**WebPack Template**
-it can be used for projects that use webpack bundling.
+# 📖 Module 2 Capstone Pokemon< <a name="about-project"></a>
 
-## Instructions <a name="Instructions"></a>
+**Module 2 Capstone Pokemon<e**
+In this project I will be using webpack to bundle my code, the project is about using multiple APIs to show and manipulate data. Styling is done with sass. Jest testing is used for the project. It will include mainly JS, HTML, Sass, and JSON.
 
-You will need to install the node_modules so the package.json and webpack.config work.
-copy the following commands in sequence:
+## 🚀 Live Demo <a name="live-demo"></a>
```

Comment:
- [ ] Please modify the live demo link of your app because it's referring to an old project (Leaderboard). Or, you might remove its section entirely for a professional README file and repo. 🙏